tests/integration/sagemaker/test_sagemaker.py

```diff
@@ -0,0 +1,55 @@
+import os
+from contextlib import AbstractContextManager
+
+import requests
+
+from jina.orchestrate.pods import Pod
+from jina.parsers import set_pod_parser
+
+
+class chdir(AbstractContextManager):
+    def __init__(self, path):
+        self.path = path
+        self._old_cwd = []
+
+    def __enter__(self):
+        self._old_cwd.append(os.getcwd())
+        os.chdir(self.path)
+
+    def __exit__(self, *excinfo):
+        os.chdir(self._old_cwd.pop())
+
+
+def test_provider_sagemaker():
```

Comment:
u can also try with a Deployment(provider) ...

And try to access it, it can be useful flr testing later.

Reply:
You sbould add a warnkng that port will be ignored

Reply:
Done